src/JustEat.StatsD/StatsDMessageFormatter.cs

```diff
@@ -10,22 +10,19 @@ namespace JustEat.StatsD
 #endif
     public class StatsDMessageFormatter
     {
-        public const string SafeDefaultIsoCultureId = "en-US";
         private const double DefaultSampleRate = 1.0;
 
-        [ThreadStatic] private static Random _random;
+        [ThreadStatic]
+        private static Random _random;
 
         private readonly CultureInfo _cultureInfo;
         private readonly string _prefix;
 
-        public StatsDMessageFormatter() : this(new CultureInfo(SafeDefaultIsoCultureId), prefix: "") {}
-
-        public StatsDMessageFormatter(string prefix = "") : this(new CultureInfo(SafeDefaultIsoCultureId), prefix) {}
-
         public StatsDMessageFormatter(CultureInfo ci, string prefix = "")
```

Comment:
We can add all the constructor overloads back in, though IMHO that's not necessary as this class is mostly used in the internals of the lib, and in the tests, so client code calling the other constructors will be extremely rare.